client/src/plus/ai-help/index.tsx

```diff
@@ -290,7 +290,8 @@ function AIHelpAssistantResponse({
 
   const isOffTopic =
     message.role === MessageRole.Assistant &&
-    message.content?.startsWith(SORRY_BACKEND_PREFIX);
+    (message.content?.startsWith(SORRY_BACKEND_PREFIX) ||
+      (message.content === "" && message.status === MessageStatus.Complete));
```

Comment:
```suggestion
    (message.content?.startsWith(SORRY_BACKEND_PREFIX) ||
      (message.status === MessageStatus.Complete &&
        (message.content === "" ||
          SORRY_BACKEND_PREFIX.startsWith(message.content))));
```

Wouldn't this be better?

Reply:
Good catch, indeed I hadn't pushed my latest changes.

Note that `SORRY_BACKEND_PREFIX.startsWith(message.content)` is always true if `message.content === """`.

Reply:
Stage has the right changes already though. I probably forgot to push and then took care of `next` (merging the local branch into it).